src/components/App.js

```diff
@@ -13,16 +13,32 @@ const App = () => {
   ]
 
   const [todos, setTodos] = useState(todosList)
+  const [input, setInput] = useState('')
+
+  const submitItem = (event) => {
```

Comment:
Un nombre un poco más específico podría ser `addTodo`. `submitItem` es muy genérico, y no expresa claramente qué es el ítem que se agrega.

```suggestion
  const addTodo = (event) => {
```